### ISSUE TITLE
Chapter 2: Installing the cog package

### ISSUE THREAD

**Issue by VictorMarchenkov** [EngineerKamesh/igb#6]
Command: go get -u github.com/uxtoolkit/cog
Returns: github.com\uxtoolkit\cog\uxcog.go:27:19: undefined: isokit.TemplateSet

**Comment by dxps** [comment]
Both `isokit` and `cog` have a new home and these should be installed using:
```shell
go get -u go.isomorphicgo.org/go/isokit
go get -u go.isomorphicgo.org/uxtoolkit/cog
```

**Comment by VictorMarchenkov** [comment]
And now new problem:
when we tried launch: gopherjs build
cannot find package "github.com/tdewolff/parse/v2" in any of:
	/usr/local/Cellar/go/1.11.2/libexec/src/github.com/tdewolff/parse/v2 (from $GOROOT)
	/Users/victor/GOPROJECTS/src/github.com/tdewolff/parse/v2 (from $GOPATH)

**Comment by dxps** [comment]
Yeah. And that's because GopherJS does not support Go modules yet.
Please find [here](https://github.com/vision8tech/vision8tech.github.io/tree/master/notes/isomorphic-go) my notes related to this issues.

I've also told to Kamesh that Errata should include these details for any new user.

**Comment by VictorMarchenkov** [comment]
And examples from Kamesh's book are not valid now?

**Comment by dxps** [comment]
The examples are completely valid and working. The aforementioned issues are related to the upcoming Go modules and some updates on the dependencies (minify and parse) that happend lately.

I garantee that you will successfully finish chapter 2 with a fully workable example (a locally installed webapp), same as the book is describing. ☺️
And that is the major part, the rest of the book provides all the details about design, decisions and techniques used for getting to this result.
So, keep going (as I do also) and enjoy the journey!

I could update my notes with everything that I did different (like Redis version, I installed v 5.0.2 instead if 4.0.2).

**Comment by VictorMarchenkov** [comment]
Many thanks for your support!

**Comment by surinkim** [comment]
@VictorMarchenkov 
If your problem is resolved, why not close this issue?
